clang/runtime/dpct-rt/include/kernel.hpp.inc

```diff
@@ -86,5 +86,15 @@ static kernel_function_info get_kernel_function_info(const void *function) {
 }
 // DPCT_LABEL_END
 
+// DPCT_LABEL_BEGIN|dpct_export_define|dpct
+// DPCT_DEPENDENCY_EMPTY
+// DPCT_CODE
+#ifdef _WIN32
+#define DPCT_EXPORT __declspec(dllexport)
```

Comment:
Could we move DPCT_EXPORT to util.hpp so that we only have one DPCT_LABEL_BEGIN in this commit?  I don't think having such fine-grained header analysis is beneficial (i.e. args_selector and dpct_export_define will always or almost always be used together).